src/store/index.ts

```diff
@@ -0,0 +1,146 @@
+import { createStore } from 'vuex';
+import axios from 'axios';
+import router from '../router';
+import { Post, AppState } from '../store/types';
+
+const BACKEND_URL = 'http://localhost:3000';
+
+const store = createStore<AppState>({
+  state: {
+    posts: [],
+    showDeleteAlert: false,
+    showCreateAlert: false,
+    selectedPost: {
+      id: '',
+      image: '',
+      selectedTags: [],
```

Comment:
The API uses `tags` instead of `selectedTags` which makes updating a post not working as expected. You need to update the tags of the post but here you are instead adding a new field called `selectedTags`.